TestTools/StreamChatTestTools/Extensions/Unique/URL+Unique.swift

```diff
@@ -26,7 +26,13 @@ public extension URL {
 
 extension URL {
     static var temporaryDirectoryRoot: URL {
-        URL(fileURLWithPath: NSTemporaryDirectory(), isDirectory: true)
+        try! FileManager.default.url(
+            for: .applicationSupportDirectory,
```

Comment:
how long the data was kept in the app support directory? We could loose the temp directory benefits with this change?

Reply:
I changed it to caches but not sure about iOS cleanup policy for it.